test/integration/framework/src/siftool/sifchain.py

```diff
@@ -535,6 +539,32 @@ def validate_genesis(self):
         res = exactly_one(stdout(res).splitlines())
         assert res.endswith(" is a valid genesis file")
 
+    # Pause the ethbridge module's Lock/Burn on an evm_network_descriptor
+    def pause_peggy_bridge(self, admin_account_address) -> List[Mapping[str, Any]]:
```

Comment:
There is already a `JsonDict` type for this purpose.

Reply:
which creates a dictionary in runtime right? The whole point in having a type is that we can explicitly know what members will belong in the result for use. It could only have the fields we need but it should not be magic like `res[0]['code']`